lib_src/src/fixed_factor_of_3_voice/src_ff3v_fir_generator.py

```diff
@@ -29,11 +29,12 @@ def plot_response_passband(fs, w, h, title):
 
 def generate_header_file(num_taps_per_phase, num_phases):
     header_template = """\
-
-/* This file is generated using src_ff3v_fir_generator.py
- *
- * Do not modify it directly, changes may be overwritten!
- */
+// Copyright (c) 2016-2017, XMOS Ltd, All rights reserved
```

Comment:
The date range in lib_src/src/fixed_factor_of_3_voice/src_ff3v_fir_generator.py is very likely to up being incorrect. 

The xmos_source_check script adds the correct date ranges to all .h, .c, and .xc files so I'd suggest undoing this change myself.